src/coreclr/src/jit/emitxarch.cpp

```diff
@@ -5810,6 +5807,7 @@ void emitter::emitIns_SIMD_R_R_S(
     }
 }
 
+#ifdef FEATURE_HW_INTRINSICS
```

Comment:
I only made the minimum number of `emitIns_SIMD_*` methods needed available outside `FEATURE_HW_INTRINSICS` as the others can't be encountered during normal codegen (and likely never will be).